.github/workflows/companion-deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Companion Deploy
+on:
+  push:
+    branches: [master]
+
+jobs:
+  docker:
+    name: DockerHub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Log in to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{secrets.DOCKER_USERNAME}}
+          password: ${{secrets.DOCKER_PASSWORD}}
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: packages/@uppy/companion
+          file: packages/@uppy/companion/Dockerfile
+          # TODO add a companion version tag if this is a "Release" commit
```

Comment:
I think there was a fix for it a few months ago that made releases work. I could add it if you're cool with that :pray:

Reply:
that'd be great :) I'm not sure if this docker-build-push action I'm using is flexible enough for this, maybe we need to go back to a bash script, but either is fine